auditors/nonroot/nonroot.go

```diff
@@ -127,3 +162,15 @@ func isContainerRunAsNonRootCSCFalse(container *k8stypes.ContainerV1) bool {
 func isContainerRunAsNonRootNil(container *k8stypes.ContainerV1) bool {
 	return container.SecurityContext == nil || container.SecurityContext.RunAsNonRoot == nil
 }
+
+func isContainerRunAsUserNil(container *k8stypes.ContainerV1) bool {
+	return container.SecurityContext == nil || container.SecurityContext.RunAsUser == nil
+}
+
+func isPodRunAsUserNil(podSpec *k8stypes.PodSpecV1) bool {
+	if podSpec.SecurityContext == nil || podSpec.SecurityContext.RunAsUser == nil {
```

Comment:
can we not do the same as you did for `isContainerRunAsUserNil` here?

`return podSpec.SecurityContext == nil || podSpec.SecurityContext.RunAsUser == nil`